pkgs/applications/misc/pe-bear/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, qtbase
+, wrapQtAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pe-bear";
+  version = "0.6.1";
+
+  src = fetchFromGitHub {
+    owner = "hasherezade";
+    repo = "pe-bear";
+    rev = "v${version}";
+    sha256 = "jzgsjqic5rBsyuwJW9T44rKM8rKDce564VAogDvsLho=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    qtbase
+  ];
+
+  meta = with lib; {
+    description = "Portable Executable reversing tool with a friendly GUI";
+    homepage = "https://hshrzd.wordpress.com/pe-bear/";
+    license = licenses.gpl2Only;
```

Comment:
`pe-bear` is apparently based on capstone & https://github.com/hasherezade/bearparser/blob/master/LICENSE
as those are vendored dependencies, I think this should be `[ (pe-bear license) (capstone license) (bearparser license) ]` but I don't recall if there is a precedent for that.

Reply:
I've added the licenses of the vendored libs.